pulp_service/pulp_service/app/tasks/package_scan.py

```diff
@@ -0,0 +1,111 @@
+import aiohttp
+import logging
+import json
+
+from asgiref.sync import sync_to_async
+
+from pulpcore.app.models import Content, RepositoryVersion
```

Comment:
It's not safe to import directly from pulpcore.app. Any plugin importing from pulpcore should only import from `pulpcore.plugin` aka the "Plugin API" which is guaranteed stable except for "plugin API breaking change releases", e.g. 3.55, 3.70, 3.85, etc. You'll find all of these objects available in `pulpcore.plugin` See [here](https://github.com/pulp/pulpcore/blob/main/pulpcore/plugin/models/__init__.py).